.python-version

```diff
@@ -1 +1 @@
-3.8.10
+3.8
```

Comment:
We run 3.8.10 in the containers but circleci has [only newer 3.8.18 available](https://circleci.com/developer/images/image/cimg/python). It refuses to run python if the version prefix does not match. This is fine as those are static checks only.